Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Download/use only archives we really need.
Don't install unnecessary stuff.

--- apache-tomcat-9.0.111-src/build.xml
+++ apache-tomcat-9.0.111-src/build.xml
@@ -1502,8 +1502,6 @@ </target>
   <target name="deploy" depends="package,build-docs,build-tomcat-jdbc,compile-webapp-examples"
           description="Default. Builds a working Tomcat instance">
 
-    <copy tofile="${tomcat.build}/bin/commons-daemon.jar" file="${commons-daemon.jar}" />
-
     <!-- Copy scripts -->
     <copy todir="${tomcat.build}/bin">
       <fileset dir="bin">
@@ -2255,12 +2253,6 @@ <include name="**/*.dll"/>
       </fileset>
     </copy>
 
-    <!-- Source bundles for native components -->
-    <copy tofile="${tomcat.dist}/bin/tomcat-native.tar.gz"
-            file="${tomcat-native.tar.gz}" />
-    <copy tofile="${tomcat.dist}/bin/commons-daemon-native.tar.gz"
-            file="${commons-daemon.native.src.tgz}" />
-
     <!-- platform README files -->
     <echo append="false" file="${tomcat.dist}/bin/x64/README">
 Apache Tomcat ${version} native binaries for Win64 AMD64/EMT64 platform.
@@ -3566,16 +3558,6 @@ <target name="download-compile"
           description="Download components necessary to compile"
           depends="setup-bnd">
 
-    <!-- Download Commons Daemon -->
-    <antcall target="downloadgz-2">
-      <param name="sourcefile.1" value="${commons-daemon.bin.loc.1}"/>
-      <param name="sourcefile.2" value="${commons-daemon.bin.loc.2}"/>
-      <param name="destfile" value="${commons-daemon.jar}"/>
-      <param name="checksum.enabled" value="${commons-daemon.bin.checksum.enabled}"/>
-      <param name="checksum.algorithm" value="${commons-daemon.bin.checksum.algorithm}"/>
-      <param name="checksum.value" value="${commons-daemon.bin.checksum.value}"/>
-    </antcall>
-
     <!-- Download JDT (Eclipse compiler) -->
     <antcall target="downloadfile-2">
       <param name="sourcefile.1" value="${jdt.loc.1}"/>
@@ -3738,16 +3720,6 @@ </target>
   <target name="download-dist"
           description="Download additional components for a distribution" >
 
-    <antcall target="downloadfile-2">
-      <param name="sourcefile.1" value="${tomcat-native.loc.1}"/>
-      <param name="sourcefile.2" value="${tomcat-native.loc.2}"/>
-      <param name="destfile" value="${tomcat-native.tar.gz}"/>
-      <param name="destdir" value="${tomcat-native.home}"/>
-      <param name="checksum.enabled" value="${tomcat-native.src.checksum.enabled}"/>
-      <param name="checksum.algorithm" value="${tomcat-native.src.checksum.algorithm}"/>
-      <param name="checksum.value" value="${tomcat-native.src.checksum.value}"/>
-    </antcall>
-
     <antcall target="downloadzip-2">
       <param name="sourcefile.1" value="${tomcat-native.win.1}"/>
       <param name="sourcefile.2" value="${tomcat-native.win.2}"/>
@@ -3758,16 +3730,6 @@ <param name="checksum.algorithm" value="
       <param name="checksum.value" value="${tomcat-native.win.checksum.value}"/>
     </antcall>
 
-    <antcall target="downloadfile-2">
-      <param name="sourcefile.1" value="${commons-daemon.native.src.loc.1}"/>
-      <param name="sourcefile.2" value="${commons-daemon.native.src.loc.2}"/>
-      <param name="destfile" value="${commons-daemon.native.src.tgz}"/>
-      <param name="destdir" value="${commons-daemon.home}"/>
-      <param name="checksum.enabled" value="${commons-daemon.native.src.checksum.enabled}"/>
-      <param name="checksum.algorithm" value="${commons-daemon.native.src.checksum.algorithm}"/>
-      <param name="checksum.value" value="${commons-daemon.native.src.checksum.value}"/>
-    </antcall>
-
     <antcall target="downloadzip-2">
       <param name="sourcefile.1" value="${commons-daemon.native.win.loc.1}"/>
       <param name="sourcefile.2" value="${commons-daemon.native.win.loc.2}"/>
